src/server/tiered_storage.h

```diff
@@ -92,12 +97,17 @@ class TieredStorage {
   struct PerDb;
   std::vector<PerDb*> db_arr_;
 
-  absl::flat_hash_map<uint32_t, uint8_t> page_refcnt_;
+  // absl::flat_hash_map<uint32_t, uint8_t> page_refcnt_;
+
+  absl::flat_hash_map<uint32_t, std::pair<unsigned, unsigned> > page_refcnt_;
```

Comment:
I suggest using a struct instead of std::pair<unsigned, unsigned>
it will make the code more clear what is first and what is second here

Reply:
this is too simple and it is not necessary in my opinion. But I added a comment above this type. That shall clarify everything when somebody wants to change it.